bin/grep

```diff
@@ -144,14 +144,14 @@ sub parse_args {
 
 	if (defined $opt{'f'}) {           # -f patfile
 		die("$Me: $opt{f}: is a directory\n") if ( -d $opt{f} );
-		open PATFILE, '<', $opt{f} or die qq($Me: Can't open '$opt{f}': $!);
+		open PATFILE, '<', $opt{f} or die qq($Me: Can't open '$opt{f}': $!\n);
```

Comment:
Not a big deal and I know you've been doing this elsewhere, but we can update these to lexical file handles when we run across them.

This doesn't block this patch, but I've just been in a vibe of not letting problems go past me at work, which is why my current task at work looks like a big PR and is taking too long. :)